### ISSUE TITLE
What are the values in the normalized x and y coordinates in SVHN -> MNIST model?

### ISSUE THREAD

**Issue by hsm207** [mingyuliutw/UNIT#39]
In Appendix B, in the paragraph about SVHN -> MNIST, it is written:

>  For each input image, we created a 5-channel variant where the first three channels were the original RGB images and the last two channels were the normalized x and y coordinates.

It seems to me that this is implement in cocogan_trainer_da.py:

```
  def _create_xy_image(self, width=32):
    coordinates = list(itertools.product(range(width), range(width)))
    arr = (np.reshape(np.asarray(coordinates), newshape=[width, width, 2]) - width/2 ) / (width/2)
    new_map = np.transpose(np.float32(arr), [2, 0, 1])
    xy = Variable(torch.from_numpy(new_map), requires_grad=False)
    return xy
```

The way the code is written, the contents of `xy` is just -1 or 0 because the environment is Python 2. Did you mean to do this? I thought the values should be between -1 and 1.

**Comment by mingyuliutw** [comment]
Thanks for reporting. This is a bug. The xy image should be in [-1 1]. I have pushed the update.
